tests/unit/test_basic.c

```diff
@@ -172,9 +172,6 @@ SENTRY_TEST(crashed_last_run)
         SENTRY_TEST_OPTIONS_NEW(options);
         TEST_CHECK(sentry__clear_crash_marker(options));
         sentry_options_free(options);
-
-        // -1 before sentry_init()
-        TEST_CHECK_INT_EQUAL(sentry_get_crashed_last_run(), -1);
```

Comment:
Note: this was actually broken and only ever passed because unit-tests in this repo normally don't run as a single invocation with all tests but individual executions launched from a python script - one execution for each test - so that when the static variable was accessed here, it was -1.